bigtable/google/cloud/bigtable/instance.py

```diff
@@ -29,8 +29,16 @@
 _EXISTING_INSTANCE_LOCATION_ID = 'see-existing-cluster'
 _INSTANCE_NAME_RE = re.compile(r'^projects/(?P<project>[^/]+)/'
                                r'instances/(?P<instance_id>[a-z][-a-z0-9]*)$')
+
+INSTANCE_TYPE_PRODUCTION = enums.Instance.Type.PRODUCTION
+INSTANCE_TYPE_DEVELOPMENT = enums.Instance.Type.DEVELOPMENT
+INSTANCE_TYPE_UNSPECIFIED = enums.Instance.Type.TYPE_UNSPECIFIED
+
 ROUTING_POLICY_TYPE_ANY = 1
 ROUTING_POLICY_TYPE_SINGLE = 2
+
+STORAGE_TYPE_SSD = enums.StorageType.SSD
+STORAGE_TYPE_HDD = enums.StorageType.HDD
 _STORAGE_TYPE_UNSPECIFIED = enums.StorageType.STORAGE_TYPE_UNSPECIFIED
```

Comment:
Likewise with storage types in the `enums` module.

Reply:
Fixed